custom_components/powercalc/data/sonos/sub/model.json

```diff
@@ -0,0 +1,26 @@
+{
+    "measure_description": "Streamed Underworld - Dark & Long for this test",
```

Comment:
What's the reason why you did not use pink noise, which is done for all other measurements of the smart speakers.
As it's a more scientific approach to getting average consumption and reproducable results.

Reply:
Since this is a sub woofer it only goes up to 80Hz (by default) and the pink noise is mostly above that. 
I tried with pink noise but the power usage was way less and did match to use case in my mind :)

Reply:
Ah I understand. seems reasonable. Cool song btw!